main.go

```diff
@@ -790,6 +858,25 @@ func configureSite(ctx *cli.Context, siteNum int, siteStrs []string, healthCheck
 	var backends []*Backend
 	var prevScheme string
 	var transport http.RoundTripper
+	var connStats []*ConnStats
+	var hostName string
+	if len(endpoints) == 1 && ctx.GlobalBool("rr-dns-mode") {
```

Comment:
We should print a log indicating that we are moving into `rr-dns-mode` instead of silently doing it.